web-ui/src/components/guild-results/GuildResults.jsx

```diff
@@ -49,7 +82,34 @@ const GuildResults = () => {
             setSearchText(e.target.value);
           }}
         />
-        <GuildsActions />
```

Comment:
This `GuildActions` component was very small and only used here. To simplify communication between that code and this code, I copied the code into here and deleted that component.